org.adempiere.install/src/org/compiere/install/ConfigurationData.java

```diff
@@ -860,6 +915,25 @@ public boolean save()
 				System.err.println(ConfigurationPanel.res.getString("ErrorSave"));
 			return false;
 		}
+		finally
+		{
+			if (fos != null) {
+				try {
+					fos.flush();
+					fos.close();
```

Comment:
this is a bad pattern that defeat the purpose of having close in finally. you want to ensure close is try regardless whether flush is throwing exception.